### ISSUE TITLE
Question regarding train/val curve 

### ISSUE THREAD

**Issue by LeoXinhaoLee** [karpathy/nanoGPT#341]
Hi, thank you very much for releasing this helpful codebase. I'm wondering if the plot shown in README is the training/val curve of GPT-2 on OpenWebText-2 from scratch. When I train the model from scratch I notice the loss starts from around 10, while the plot starts from around 3.6. 

Thank you very much!

**Comment by VatsaDev** [comment]
![image](https://github.com/karpathy/nanoGPT/assets/71975550/3c9a55ec-406b-49b2-96a5-da1706bba4ca)

While I myself haven't Trained from openwebtext from scratch, I have been training and finetuning other with datasets, and so far the highest I have seen Is a 4, and the lowest a 1.9. I'm assuming the loss of 10 is just a very early train which drops very fast, as the graph itself has a high slope in the beginning

**Comment by LeoXinhaoLee** [comment]
I see. Indeed when we train from scratch on OpenWebText, the loss starts from around 10 and quickly decreases. Thank you!
